Robot2019/src/main/java/frc/robot/commands/Climb.java

```diff
@@ -33,28 +38,30 @@ protected void initialize() {
   // Called repeatedly when this Command is scheduled to run
   @Override
   protected void execute() {
-    dt.drive(-0.5, -0.5);
-    if (up) {
-      climber.runClimber(0.5);
-      if (!climber.needToClimb())
-        up = false;
+    if (state == State.CLIMB) {
+      climb();
+      if (!dt.isStalled() || Math.abs(dtJoy.getY()) > 0.1) { // 0.1 = Placeholder/approximation 
```

Comment:
what is the joystick supposed to do?

Reply:
Checks if driver is trying to drive. Not sure what to fix.

Reply:
I'd suggest a button override instead